browser/js/dashboard/dashwrapper.html

```diff
@@ -1,11 +1,11 @@
-<div layout="row" class="dashboard-wrapper">
+<div layout="row" class="dashboard-wrapper" ng-cloak>
     <div layout="column" flex style="height: 100%">
-        <div layout="row" id="dateNav">
-            <a ui-sref="dashboard.week({week:currentWeekNum-1})" ui-sref-opts="{reload: true, inherit: false}">
+        <div ng-if="loaded" layout="row" id="dateNav">
+            <a ng-click="go(currentWeekNum-1)">
                 <i class="left-one material-icons">keyboard_arrow_left</i>
             </a>
             <span id="displayweek" ng-cloak>{{displayWeek}}</span>
-            <a ui-sref="dashboard.week({week:currentWeekNum+1})" ui-sref-opts="{reload: true, inherit: false}">
```

Comment:
Switching from the `ui-sref` means users won't be able to copy the url, nor see the url (in the bottom left of the browser window) when hovering over it. Still might make sense to keep this decision, but just thought I'd mention.